### PR TITLE
Watchlist

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds a watchlist for new photometry.

### CODE REVIEW

File: services/watch_list/watch_list.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                "objectId": 1,
+            }
+            alerts = get_alerts_by_position(
+                obj.ra,
+                obj.dec,
+                2.0,  # arcseconds
+                "arcsec",
+                program_id_selector,
+                projection=projection,
+                include_all_fields=True,
```
[the remark this answers is not in the document]
Comment (Theodlz) [reply services/watch_list/watch_list.py:95]: Oh and this method is implemented in Fritz only correct?

**Review by Theodlz — COMMENTED** [review]
As discussed on Slack, this looks fine but some constraints as too how often it runs and how much data it grabs seem necessary to avoid flooding the database with candidates and hitting Kowalski's API too hard. Just like we ask for users writing scripts to add backoffs, this needs to be a little more robust that way.

File: services/watch_list/watch_list.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Theodlz) [reply services/watch_list/watch_list.py:95]: Oh maybe we simply want this to run once a day after all alerts have been processed. So we just need to keep track of the last date at which this ran. If new_date > last_date and current time > the UTC date at which Kowalski is done ingesting, than run that and update the last_ran date.

[the remark this answers is not in the document]
Comment (mcoughlin) [reply services/watch_list/watch_list.py:95]: @Theodlz Can I ask you to take over this one to implement your vision?

[the remark this answers is not in the document]
Comment (mcoughlin) [reply services/watch_list/watch_list.py:95]: Of course. The spirit of it is basically: for a particular location on the sky (which may or may not have been active before in the alert stream), I want to get notifications as soon as there is a detection.

**Pull request merged**